src/Aspire.ProjectTemplates/templates/aspire-xunit/IntegrationTest1.cs

```diff
@@ -0,0 +1,37 @@
+using System.Net;
+
+namespace Aspire.Tests1;
+
+public class IntegrationTest1
+{
+    // Instructions:
+    // 1. Add a project reference to the target AppHost project, e.g.:
+    //
+    //    <ItemGroup>
+    //        <ProjectReference Include="../MyAspireApp.AppHost/MyAspireApp.AppHost.csproj" />
+    //    </ItemGroup>
+    //
+    // 2. Update the target AppHost project to allow this test project to see its internals, e.g.:
+    //
+    //    <ItemGroup>
+    //        <InternalsVisibleTo Include="Aspire.Tests1" />
```

Comment:
This seems... unfortunate. Is this the best we can do?

Reply:
AppHost projects don't typically have a public type in them that can be used as an entry point and this is the pattern we usually show for ASP.NET Core projects and `WebApplicationFactory`. IVT is pretty common for test scenarios but I'm open to other approaches that work with the the APIs.

Reply:
Idea: we could update our AppHost targets that generate the project metadata classes to also generate a type for the AppHost project itself and then that could be used, e.g. `DistributedApplicationTestBuilder<Projects.MyAspireApp_AppHost>`.

Reply:
Oh actually we already generate one for the AppHost project itself, but it's `internal`! Should we just change that to be `public`?

Reply:
Yeah, I think reusing this type would be ideal. We might want to add a switch that generates the type as `internal` though, just in case someone cares.

Reply:
OK this is done now.